Makefile

```diff
@@ -142,28 +142,6 @@ ci-e2e: build-scorecard check-env e2e-cron | $(GINKGO)
 	@echo Ignoring these test for ci-e2e $(IGNORED_CI_TEST)
 	ginkgo -p  -v -cover --skip=$(IGNORED_CI_TEST)  ./e2e/...
 
-test-disk-cache: ## Runs disk cache tests
```

Comment:
So will this mean customers cannot use the DISK cache? Does this use cache?

Reply:
My main aim in this PR is to simply update the API such that we can pass in a different HTTP client if/when required (I don't have a concrete solution in mind as of yet). Only reason to change the command line tool was because I thought having a cache here might be an overkill. But there are too many failing tests and I do not want to spend time fixing them.

So, I'll keep the cache everywhere for now, but change the API.